functions/src/index.ts

```diff
@@ -143,3 +145,124 @@ export const beforecreated: BlockingFunction = beforeUserCreated(
     }
   },
 )
+
+export const exportHealthSummary = onRequest(async (req, res) => {
+  const data = await generateHealthSummary({
```

Comment:
Would be great if we can create a small test that can verify the output of the function based on a pre-defined input (e.g. the one you have in the cloud function definition) to validate that we don't have any major regressions in the output.
Something like https://github.com/btravers/pdf-snapshot could get us a long way there and we could embed the example output that we expect right here in the repo.

Would also be great to include the test coverage + testing setup right in this repo as an extension to the GitHub action.

Reply:
Created this issue: https://github.com/StanfordBDHG/ENGAGE-HF-Firebase/issues/16